tests/integration/_conftest/tokenizer.py

```diff
@@ -0,0 +1,22 @@
+from typing import Any
+
+
+def tokenize(positional_args: list[Any], named_args: dict[str, Any]):
+    tokens: list[str] = []
+    for value in positional_args:
+        if value is not None:
+            tokens.append(str(value))
+    for name, value in named_args.items():
+        if value is None:
+            continue
+        if isinstance(value, bool):
+            if value:
+                tokens.append(f"--{name}")
+            continue
```

Comment:
```suggestion
        if value == False:
            continue
```

Reply:
It won't work because if the value is True, the code below will append key and value instead of key only. Argparse expects no values for flags.

Reply:
Ah, right. Nevertheless, I think you can still skip the `isinstance` thing and just check strictly for `True` to avoid an implicit casting:
```
if value == True:
  tokens.append(f"--{name}")
  continue
```

Reply:
You need an explicit check for "False" then.

I changed it, as you propose, to avoid nested conditions.